plugins/generic/dataverse/DataversePlugin.inc.php

```diff
@@ -1171,7 +1171,7 @@ function createMetadataPackage($article) {
 		$package->addMetadata('description', 
 						$article->getLocalizedData('studyDescription') ? 
 						$article->getLocalizedData('studyDescription') :
-						$article->getLocalizedAbstract()
+						String::html2text($article->getLocalizedAbstract())
```

Comment:
By the way, see http://pkp.sfu.ca/bugzilla/show_bug.cgi?id=9056 -- deposits generally shouldn't be using the getLocalized... functions because those are dependent on the current user's locale, which means a depositor working e.g. in English will deposit different content from someone working e.g. in Spanish.

Better to use, for journal-level content, e.g...
    $journal->getLocalizedSetting($journal->getPrimaryLocale())

For article-level content, e.g.:
    $article->getTitle($article->getLocale())